README.md

```diff
@@ -111,5 +111,9 @@ New `wasm-code` binary file will be generated in the `runtime` folder.
 1. Go to `./sandbox`
 2. Execute `bash run-it.sh`
 
+### 5. Run the Unit Tests for Spec-Builder
+1. Go to `./runtime/tools/spec-builder`
```

Comment:
I think that you need to add info about the building of the wasm module. Maybe it would be best to simply write: 
```
### 5. Running the Unit Tests for Spec-Builder
0. Build the `wasm-module` used for the tool (read instructions in the spec-builder README)
1.  Go to `./runtime/tools/spec-builder`
2. Execute `yarn run test`
```